plugins/network-elements/nuage-vsp/src/com/cloud/network/resource/NuageVspResource.java

```diff
@@ -262,15 +260,19 @@ public String getName() {
     public PingCommand getCurrentStatus(long id) {
         if ((_relativePath == null) || (_relativePath.isEmpty()) || (_cmsUserInfo == null) || (_cmsUserInfo.length == 0)) {
             s_logger.error("Failed to ping to Nuage VSD");
+            _shouldAudit = true;
             return null;
         }
         try {
             login();
         } catch (Exception e) {
             s_logger.error("Failed to ping to Nuage VSD on " + _name + " as user " + _cmsUserInfo[1] + " Exception " + e.getMessage());
```

Comment:
During the configuration of this resource, the value of _cmsUserInfo is set to `_cmsUserInfo = new String[] {CMS_USER_ENTEPRISE_NAME, cmsUser, cmsUserPass};`
Therefore, this is safe :)